src/test/java/uk/gov/companieshouse/personapi/controller/PersonControllerTest.java

```diff
@@ -15,28 +23,24 @@
 import uk.gov.companieshouse.personapi.service.PersonService;
 
 import java.time.LocalDate;
-import java.time.LocalTime;
-
-import static org.hamcrest.Matchers.any;
-import static org.hamcrest.Matchers.is;
-
-import static org.mockito.Mockito.when;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 @ExtendWith(SpringExtension.class)
 @WebMvcTest(controllers = PersonController.class)
 public class PersonControllerTest {
     private static final String PERSON_ID = "abcd";
     private static final String ERROR_MESSAGE = "Test successful";
+
     @Autowired
     private MockMvc mvc;
     @MockBean
     private PersonService personService;
+
     private Address address;
     private Person person;
+    private Person obj;public List<Object> people;
```

Comment:
make this List <Person> as <object> is redundant

Reply:
I think using `<` and `>` has made your comment format weirdly.
the original comment should say: "make this `List<Person>` as `List<Object>` is reduntant"